appinventor/blocklyeditor/src/exportBlocksImage.js

```diff
@@ -376,12 +377,18 @@ PNG.prototype.processData_ = function(data) {
     return data.slice(chunkStart, chunkStart + length);
   }
   this.chunks = [];
+  var phy = [112, 72, 89, 115];
```

Comment:
I still don't like doing this in `processData_` because it is used on multiple code paths, not just in the PNG generation. Instead, I'd prefer if we move the logic of adding the chunk to the `download` function below, similar to how `setCodeChunk` is handled. Lastly, we should also check to see if there is a phys chunk and replace it. At the moment, browsers don't appear to be adding the chunk, but in the off chance they do so in the future we want to replace the existing chunk rather than have 2 in conflict.